src/index.html

```diff
@@ -10,13 +10,109 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>MOYO header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+      <img src="images/logo.png" alt="MOYO company logo" class="logo" />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="#apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#samsung"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#smartphones"
+              class="nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#laptops&computers"
```

Comment:
Be careful with using special characters like '&' in the href attribute of the <a> tag. It's better to avoid them to prevent any potential issues. You could replace '&' with 'and'.